.github/workflows/tool-cleanup.yml

```diff
@@ -0,0 +1,43 @@
+# Copyright 2023 The Authors (see AUTHORS file)
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#      http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Workflow that handles AOD tool requests cleanup.
+name: 'aod-tool-cleanup'
```

Comment:
Just name it `aod-cleanup` and file as `cleanup.yml`. Like I'm proposing offline, we should merge in the IAM cleanup.

Reply:
Since currently we only supports tool cleanup, we can keep it until we supports iam cleanup?